.circleci/config.yml

```diff
@@ -30,29 +30,6 @@ executors:
           - network.host: 127.0.0.1
           - http.port: 9200
           - discovery.type: single-node
-  accessibility_test_exec:
```

Comment:
This executor was a duplicate of `integration_test_exec`, so deleted this and used `integration_test_exec` instead